openedx_tagging/core/tagging/models/base.py

```diff
@@ -137,7 +138,7 @@ class Taxonomy(models.Model):
         ),
     )
     allow_multiple = models.BooleanField(
-        default=False,
+        default=False,  # TODO: This should be true, or perhaps remove this property altogether
```

Comment:
@ChrisChV @pomegranited Do you know if we actually need this `allow_multiple` flag? I don't see any requirement for it in the UX mockups etc.

Same question about `required`

Reply:
I think we need `allow_multiple`, but `required` can be removed.
E.g. we only want one Language tag per content item, but we'll want to allow multiple LightCast tags to be added.

Reply:
I agree with @pomegranited :+1: